packages/dataviews/src/types.ts

```diff
@@ -131,6 +131,11 @@ export type Field< Item > = {
 	 */
 	description?: string;
```

Comment:
Since we're keeping this prop in the interface (which makes sense to me, since we probably don't want this to be a breaking change, to ease upstreaming) then we should probably add a `@deprecated` annotation.